addons/fire/CfgSounds.hpp

```diff
@@ -0,0 +1,25 @@
+#define CONCAT(a,b) a##b
```

Comment:
pboProj complains
`In File P:\z\ace\addons\fire\CfgSounds.hpp: circa Line 1 #Define bad label`

Reply:
`#define CONCAT(a,b) a####b` works, no clue why